sdformat_mjcf/src/sdformat_mjcf/mjcf_to_sdformat/converters/geometry.py

```diff
@@ -154,4 +154,6 @@ def mjcf_collision_to_sdf(geom):
         col.set_geometry(sdf_geometry)
     else:
         return None
+    if geom.friction is not None:
+        col.surface().friction().ode().set_mu(geom.friction[0])
```

Comment:
If the friction is None, we should set to the default value of 1.
```suggestion
    friction = su.get_value_or_default(geom.friction, [1, 0.005, 0.0001])
    col.surface().friction().ode().set_mu(friction[0])
```

Reply:
https://github.com/gazebosim/gz-mujoco/pull/86/commits/21eee33d47b645e196b37a7b5df99eb3b2975968